src/cosmos/tx/messages/register.rs

```diff
@@ -0,0 +1,35 @@
+//! Register transaction message
+
+use actix::Message;
+use cosmrs::tx::Msg;
+use serenity::model::user::User;
+
+/// Result of a register tx message.
+pub type RegisterTxResult = ();
+
+/// Register transaction actor message.
+#[derive(Message)]
+#[rtype(result = "RegisterTxResult")]
+pub struct RegisterTx<T>
```

Comment:
That's not a transaction 😉 
```suggestion
pub struct RegisterMsg<T>
```